cfgov/v1/models/base.py

```diff
@@ -126,6 +128,9 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
+    def clean(self):
```

Comment:
Does this method need to call the `super` clean, to validate against any base Wagtail validators (like [this code](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailcore/models.py#L456), that validates that the slug is unique)? This doesn't appear to cause problems via the UI (because it has additional checks via the Django form) but skipping that might allow you to programmatically create invalid pages.